src/types/emr/patient/patientApi.ts

```diff
@@ -39,6 +67,97 @@ export default {
       }>
     >(),
   },
+
+  // Questionnaire Responses
+  getQuestionnaireResponses: {
+    path: "/api/v1/patient/{patientId}/questionnaire_response/",
+    method: HttpMethod.GET,
+    TRes: Type<PaginatedResponse<QuestionnaireResponse>>(),
+  },
+  getQuestionnaireResponse: {
+    path: "/api/v1/patient/{patientId}/questionnaire_response/{responseId}/",
+    method: HttpMethod.GET,
+    TRes: Type<QuestionnaireResponse>(),
+  },
+
+  // Observations
+  listObservations: {
+    path: "/api/v1/patient/{patientId}/observation/",
+    method: HttpMethod.GET,
+    TRes: Type<PaginatedResponse<Observation>>(),
+  },
+  observationsAnalyse: {
+    path: "/api/v1/patient/{patientId}/observation/analyse/",
+    method: HttpMethod.POST,
+    TRes: Type<ObservationAnalyzeResponse>(),
+  },
+
+  // Notes and Threads
+  listThreads: {
+    path: "/api/v1/patient/{patientId}/thread/",
+    method: HttpMethod.GET,
+    TRes: Type<PaginatedResponse<Thread>>(),
+    TQuery: Type<{ encounter?: string }>(),
+  },
+  createThread: {
+    path: "/api/v1/patient/{patientId}/thread/",
+    method: HttpMethod.POST,
+    TRes: Type<Thread>(),
+    TBody: Type<{ title: string; encounter?: string }>(),
+  },
+  getMessages: {
+    path: "/api/v1/patient/{patientId}/thread/{threadId}/note/",
+    method: HttpMethod.GET,
+    TRes: Type<PaginatedResponse<Message>>(),
+  },
+  postMessage: {
+    path: "/api/v1/patient/{patientId}/thread/{threadId}/note/",
+    method: HttpMethod.POST,
+    TRes: Type<Message>(),
+    TBody: Type<{ message: string }>(),
+  },
+
+  // User Management
+  addUser: {
+    path: "/api/v1/patient/{patientId}/add_user/",
+    method: HttpMethod.POST,
+    TRes: Type<UserBase>(),
+    TBody: Type<{ user: string; role: string }>(),
+  },
+  listUsers: {
+    path: "/api/v1/patient/{patientId}/get_users/",
+    method: HttpMethod.GET,
+    TRes: Type<PaginatedResponse<UserBase>>(),
+  },
+  removeUser: {
+    path: "/api/v1/patient/{patientId}/delete_user/",
+    method: HttpMethod.POST,
+    TRes: Type<{ user: string }>(),
+  },
+
+  // OTP Patient Routes
+  otpGetPatient: {
+    path: "/api/v1/otp/patient/",
+    method: HttpMethod.GET,
+    TRes: Type<PaginatedResponse<Patient>>(),
+    auth: {
+      key: "Authorization",
+      value: "Bearer {token}",
+      type: "header",
+    },
+  },
+  otpCreatePatient: {
```

Comment:
OTP Based APIs ought to be separately managed, let's not keep them together with Patient APIs.

This has been split as separate issues as well